dev-client/src/components/DataInputSummary.tsx

```diff
@@ -51,7 +51,7 @@ export const DataInputSummary = ({
           />
         )}
       </Box>
-      <Text variant="body1" fontWeight={700}>
+      <Text variant="body1" fontWeight={700} textTransform={'uppercase'}>
```

Comment:
```suggestion
      <Text variant="body1" fontWeight="700" textTransform="uppercase">
```

Reply:
second change is good style and i will make it (i'm surprised prettier doesn't fix that), the first change is not a style change, it changes the type of 700 from number to string